Doc/library/stdtypes.rst

```diff
@@ -319,7 +319,9 @@ the operations, see :ref:`operator-summary`):
 Notes:
 
 (1)
-   Also referred to as integer division.  The resultant value is a whole
+   Also referred to as integer division.  For operands of type :class:`int`,
+   the result has type :class:`int`.  For operands of type :class:`float`,
+   the result has type :class:`float`.  In general, the result is a whole
```

Comment:
```suggestion
   Also referred to as integer division.  If both operands of type :class:`int`,
   the result has type :class:`int`.  If any operand of type :class:`float`,
   the result has type :class:`float`.  In general, the result is a whole
```

Reply:
Actually, this turns to be more complicated, if we consider all numeric types of the stdlib.

It seems, the result is an int if _both_ operands are of exact type (the Scheme numerical tower has this notion), i.e. either int or Fraction.  But if any operand is a float or a Decimal - that defines the type of the result (if one defined for such a combination).

Reply:
Let's leave details about `Fraction` and `Decimal` to the respective module documentation; I don't think this is the place to go through all the possible combinations.